probe/docker/container.go

```diff
@@ -331,7 +331,11 @@ func (c *container) GetNode(hostID string, localAddrs []net.IP) report.Node {
 		ContainerIPsWithScopes: report.MakeStringSet(ipsWithScopes...),
 	}).WithLatest(
 		ContainerState, mtime.Now(), state,
-	).WithMetrics(c.metrics())
+	).WithMetrics(
+		c.metrics(),
+	).WithParents(report.Sets{
+		"container_image": report.MakeStringSet(report.MakeContainerImageNodeID(hostID, c.container.Image)),
```

Comment:
Given that almost all the taggers/reporters need to know the host id to produce their node ids, it seems a shame not to have them also add the host id to the parents and get rid of the host tagger.  OTOH, doing it in one place guarantees all nodes will have the a host as a parent.